ui2/package.json

```diff
@@ -38,10 +39,9 @@
     ]
   },
   "devDependencies": {
-    "babel-eslint": "^10.0.2",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
-    "eslint": "^6.1.0",
+    "eslint": "^5.16.0",
```

Comment:
why?

Reply:
We did this together

Reply:
try to upgrade to the latest and see what happens

Reply:
tried, these are the requirements of react-scripts